test/unit/mdc-icon-button/mdc-icon-button-toggle.test.js

```diff
@@ -77,85 +77,38 @@ if (supportsCssVariables(window)) {
     raf.flush();
     component.destroy();
     raf.flush();
-    assert.isNotOk(root.classList.contains('mdc-ripple-upgraded'));
+    assert.isFalse(root.classList.contains('mdc-ripple-upgraded'));
     raf.restore();
   });
 }
 
 test('set/get on', () => {
   const {root, component} = setupTest();
   component.on = true;
-  assert.isOk(component.on);
+  assert.isTrue(component.on);
   assert.equal(root.getAttribute('aria-pressed'), 'true');
 
   component.on = false;
-  assert.isNotOk(component.on);
+  assert.isFalse(component.on);
   assert.equal(root.getAttribute('aria-pressed'), 'false');
 });
 
-
-test('#refreshToggleData proxies to foundation.refreshToggleData()', () => {
-  const MockIconToggleFoundation = td.constructor(MDCIconButtonToggleFoundation);
-  const root = document.createElement('i');
-  const foundation = new MockIconToggleFoundation();
-  const component = new MDCIconButtonToggle(root, foundation);
-  component.refreshToggleData();
-  td.verify(foundation.refreshToggleData());
-});
-
-test('intially set to on if root has aria-pressed=true', () => {
-  const root = bel`<button class="mdc-icon-button" aria-pressed="true"></button>`;
-  const component = new MDCIconButtonToggle(root);
-  assert.isOk(component.on);
-});
-
 test('get ripple returns a MDCRipple instance', () => {
   const {component} = setupTest();
-  assert.isOk(component.ripple instanceof MDCRipple);
+  assert.isTrue(component.ripple instanceof MDCRipple);
 });
 
 test('#adapter.addClass adds a class to the root element', () => {
   const {root, component} = setupTest();
   component.getDefaultFoundation().adapter_.addClass('foo');
-  assert.isOk(root.classList.contains('foo'));
-});
-
-test('#adapter.addClass adds a class to the inner icon element when used', () => {
-  const {root, component} = setupTest({useInnerIconElement: true});
```

Comment:
We seem to never use `useInnerIconElement` or `tabIndex` in `setupTest` anymore.

Are we losing anything by no longer supporting the inner icon element? I'm inclined to think that's mitigated by having separate DOM for each state now.